src/select.jl

```diff
@@ -0,0 +1,247 @@
+#
+# Function to perform the most important selections that are used in solute-solvent
+# analysis
+#
+export @sel_str
+
+#
+# Dictionaries of residue, atom, and element types, as originally written
+# in PDBTools.jl (at first sight, seems more general that the current)
+#
+#
+# Data for natural protein residues
+#
+Base.@kwdef struct ProteinResidue
```

Comment:
I would have a slight preference for the name `AminoAcid` for this struct, since amino acid often refers to the theoretical molecule whereas residue refers to a particular instance of it in a protein.